test/e2e/suite_test.go

```diff
@@ -118,6 +118,22 @@ var _ = ginkgo.AfterSuite(func() {
 				endLog:   "Azure Disk CSI Driver uninstalled",
 			}
 			execTestCmd([]testCmd{azurediskLog, e2eTeardown})
+
+			// install/uninstall Azure Disk CSI Driver deployment scripts test
+			installDriver := testCmd{
+				command:  "bash",
+				args:     []string{"deploy/install-driver.sh", "master", "windows,snapshot,local"},
+				startLog: "===================install Azure Disk CSI Driver deployment scripts test===================",
+				endLog:   "===================================================",
+			}
```

Comment:
Is it enough that we just test the master branch image here? The image does not include this PR.

Reply:
we only test the deployment script here. Original e2e tests already tested against the image.